app/mailers/jobseeker_mailer.rb

```diff
@@ -9,11 +9,21 @@ def confirmation_instructions(record, token, _opts = {})
 
     view_mail(
       NOTIFY_JOBSEEKER_CONFIRMATION_TEMPLATE,
-      to: @jobseeker.email,
+      to: @jobseeker.pending_reconfirmation? ? @jobseeker.unconfirmed_email : @jobseeker.email,
       subject: I18n.t("jobseeker_mailer.confirmation_instructions.subject"),
     )
   end
 
+  def email_changed(record, _opts = {})
+    @jobseeker = record
+
+    view_mail(
+      NOTIFY_JOBSEEKER_EMAIL_CHANGED_TEMPLATE,
+      to: @jobseeker.email,
+      subject: I18n.t("jobseeker_mailer.email_changed.subject"),
```

Comment:
You can use the short lookup here too, just use t(".subject") and put the translation in the right place